test/test-common.sh

```diff
@@ -143,7 +143,7 @@ checkoutput() {
 	    echo expected: \'$expected\'
 	    exit 2
 	fi
-	echo "$config_crc" | md5sum -c - 2>&1 > /dev/null
+	echo "$config_crc" | md5sum -c - >/dev/null 2>&1
```

Comment:
While I agree it as poorly coded, I do not like the fix.  Why should we hide the eventual diagnostic message from the user/developer in the first place?

If something goes wrong, the user should know about it. The correct fix would IMO look such:
```
--- a/test/test-common.sh
+++ b/test/test-common.sh
@@ -143,7 +143,7 @@ checkoutput() {
        echo expected: \'$expected\'
        exit 2
    fi
-   echo "$config_crc" | md5sum -c - 2>&1 > /dev/null
+   echo "$config_crc" | md5sum -c - > /dev/null
    if [ $? != 0 ]; then
        echo "config file $output has been altered: MD5 sum mismatch"
        exit 3
```